stories/ng2-component-lab/dropdown.component.exp.ts

```diff
@@ -23,13 +94,29 @@ export default experimentOn('DropDown')
         template: `
       <sdc-dropdown label="Disabled dropDown example" placeHolder="Please choose option" disabled="true" [options]="options"></sdc-dropdown>
     `
+    }).case('Headless and labeless DropDown', {
+        context: {
+            options: options,
+        },
+        showSource: true,
+        template: `
+        <div style="margin-bottom: 10px;">
+            <button dropdown-trigger [dropDown]="dropDown">Click to toggle!</button><span> Selected value: {{ dropDown.value }}</span>
+        </div>
+        <sdc-dropdown placeHolder="Please choose option" [headless]="true" [options]="options" #dropDown></sdc-dropdown>
```

Comment:
Instead of explicitly declare "headless" when using dropdown-trigger, consider to change this input of the dropdown from the trigger directive.